README.md

```diff
@@ -62,7 +62,17 @@ Sample playbooks and instructions on how to run the modules can be found in the
 
 ## Setup
 
-To run the Ansible modules provided in this project, you may run a containerized version or perform a full installation. The containerized version of the `oneview-ansible` modules is available in the [Docker Store](https://store.docker.com/community/images/hewlettpackardenterprise/oneview-ansible-debian). There is also a [how-to guide](https://github.com/HewlettPackard/oneview-ansible-samples/blob/master/oneview-ansible-in-container/oneview-ansible-in-container.md) with instructions on how to use the container.
+To run the Ansible modules provided in this project, you may run a containerized version or perform a full installation. The containerized version of the `oneview-ansible` modules is available in the [Docker Store](https://store.docker.com/community/images/hewlettpackardenterprise/hpe-oneview-sdk-for-ansible). 
+ ```bash
+#Download and store a local copy of terraform-provider-oneview and
```

Comment:
image name should be hpe-oneview-sdk-for-ansible

Reply:
Done